### PR TITLE
Fixes #177

### LINKED ISSUE #177: region meta in Scheduler and TinyKV has same address
https://github.com/pingcap-incubator/tinykv/blob/1df74d600cb0c5001f32d2d21d6f89e9744ee3e0/kv/test_raftstore/scheduler.go#L345-L348
From the code, it seems that region meta in Scheduler and TinyKV should be independence. 
https://github.com/pingcap-incubator/tinykv/blob/1df74d600cb0c5001f32d2d21d6f89e9744ee3e0/kv/raftstore/peer.go#L345-L352
but it send a pointer to scheduler. It will cause problem when change peer. Maybe it should be cloned first.

### PR DESCRIPTION
**Pull request by xiongjiwei**
#177

### CODE REVIEW

File: kv/raftstore/peer.go [hunk 1]
```diff
@@ -343,8 +343,13 @@ func (p *peer) Term() uint64 {
...
 func (p *peer) HeartbeatScheduler(ch chan<- worker.Task) {
+	clonedRegion := new(metapb.Region)
+	err := util.CloneMsg(p.Region(), clonedRegion)
+	if err != nil {
+		return
+	}
 	ch <- &runner.SchedulerRegionHeartbeatTask{
-		Region:          p.Region(),
+		Region:          clonedRegion,
 		Peer:            p.Meta,
```
Comment (Connor1996) [inline kv/raftstore/peer.go:353]: not sure is it okay？please check it

Comment (xiongjiwei) [reply kv/raftstore/peer.go:353]: how could I check it, should I write test for scheduler.go?
it wroks very well on my machine

Comment (Connor1996) [reply kv/raftstore/peer.go:353]: I mean the `Peer` is also a pointer, please check the code to see if there is any problem like `Region`. No need to write a test.

If no problem, I will merge this PR.

Comment (xiongjiwei) [reply kv/raftstore/peer.go:353]: it's ok cause a `Peer`'s meta will not change once created. `PendingPeers` only contains meta, too.

**Review by Connor1996 — APPROVED** [review]
LGTM

**Pull request merged**